scss/style.scss

```diff
@@ -7,6 +7,7 @@ $o-grid-ie8-rules: 'none';
 @import 'o-grid/main';
 
 @include oFontsInclude(MetricWeb, light); // body
+@include oFontsInclude(MetricWeb, bold); // headings
```

Comment:
NB: this is a proposal without much thought, but it certainly improves the page design, and makes the CTA pop out. I'd also use it in other places.